continuous_delivery/deploy_and_basics.rst

```diff
@@ -81,25 +73,22 @@ Deploy a Specific Git Tag
 **Build branch** allows you to specify to deploy an arbitrary Git branch or tag.
 
 .. important::
-   #. This should only be used with test systems [#f1]_.
-   #. The value of **Build branch** is ignored if `Docker Image ID <#run-menu-advanced>`__ is set.
```

Comment:
I'd still mention somewhere that either build branch is deployed or a docker image depending on whether such an image was specified.